docs/using-radicle/view-share.md

```diff
@@ -3,81 +3,85 @@ id: view-share
-interface](https://app.radicle.network), which helps you discover new projects hosted on the network. The web interface
-is organized by either the seed node a project is synced with or the organization it's published under, such as
-Radicle's [alt-clients](https://app.radicle.network/alt-clients.radicle.eth) organization.
+After you've pushed your project to the Radicle network, you can view it using either the Upstream desktop app or the
+web interface. They have a similar user interface and experience &mdash; the primary difference is that Upstream
+supports patches.
 
-![The Radicle web interface for viewing a published project](/img/web-interface.png)
+## View a project on Upstream
+
+If you're a delegate of the project, or you've previously [cloned](clone.md) and [pushed](push.md) code to the network,
+you'll see the project on your profile page. Click on the box to view that project.
+
+If you're looking for another Radicle user's project, you need the project URN, which you can find and copy either via
+the project's delegate or the [web interface](#view-a-project-on-the-web-interface). Click on the magnifying glass icon
+(🔍) and paste in the project URN. If you're not currently tracking this project, click the **Track** button to search
+for it on the Radicle network. Once your profile page updates, click the box to view the project.
+
+![Viewing a project in Upstream](/img/upstream_project.png)
 
 ## View a project on the web interface
 
 All projects [published](create.md) to Radicle are discoverable via the web interface at `https://app.radicle.network`.
+The web interface is organized by either the seed node a project is synced with or the organization it's published
+under, such as Radicle's [alt-clients](https://app.radicle.network/alt-clients.radicle.eth) organization.
 
 Click on one of the seed nodes and explore the list of projects hosted there. Click on any project to view its tree,
 primary README, commit history, contributors, and more.
 
 ![Viewing a specific project with the web interface](/img/web-interface_project.png)
 
-### View folders and files
-
-The project view includes a tree of files and folders. Click on folders to expand them or individual files to show their
-syntax-highlighted content and the latest relevant commit.
+## Explore a project
 
-![Viewing a project's files](/img/web-interface_files.png)
-
-### View the commit history
+Both interfaces let you explore files and the commit history. Click on folders to expand them or individual files to
+show their syntax-highlighted content and the latest relevant commit.
 
 The **X commit(s)** button displays the project's commit history, including the commit message, the committer's name,
-and the time/date of the commit.
+and the time/date of the commit. Click on any commit to view changes, such as additions and deletions, as a diff.
+
+In the web interface, click the **< >** icon to explore the state of the project at that commit.
 
```

Comment:
Maybe here we should explain how one would access the patches listing of a project.
It is the same as with commits but you would use a button with **X patche(s)**

Reply:
Done!